### ISSUE TITLE
"Connection to server is erroring" on RHEL7.2

### ISSUE THREAD

**Issue by joewood** [microsoft/vscode-cpptools#293]
VSCode 1.6.1, Extension 0.9.2

I'm seeing error below when trying to use the extension on RHEL7.2:

```
[4:18:22 PM] Connection to server is erroring. Shutting down server.
[4:18:22 PM] Connection to server is erroring. Shutting down server.
[4:18:22 PM] Connection to server is erroring. Shutting down server.
```

The extension additional files have been downloaded and installed successfully. There's not much information when I start code in verbose mode. 

Is there anything else I can do to diagnose the problem?

**Comment by sean-mcmanus** [comment]
Is your OS 32-bit? The error you're seeing occurs when our 64-bit language server process fails.

**Comment by sean-mcmanus** [comment]
RHEL7.2 may not be using glibc 2.18 (7.0 doesn't). See https://github.com/Microsoft/vscode-cpptools/issues/19 for a potential workaround.

**Comment by joewood** [comment]
Yes, this is the issue - glibc dependency. Could the extensions use **npm install** and **gyp** rather than binary install to get around these dependency issues?

**Comment by bobbrow** [comment]
Duplicate of #19

**Comment by sean-mcmanus** [comment]
Fixed with 0.14.4.
